Core/COCopier.h

```diff
@@ -11,6 +11,39 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Options to control COCopier behavior.
+ */
+typedef NS_OPTIONS(NSInteger, COCopierOptions) {
+    /**
+     * Whether copies keep using the same UUIDs in destination than original items in source.
+     *
+     * You should omit this option to implement cut/copy/paste semantics, whether source and 
+     * destination are the same or not.
+     */
+    COCopierReusesSourceUUIDs = 2,
+    /**
+     * Whether items reachable through non composite references are copied, when they dont'
+     * exist in the destination.
+     *
+     * Be careful with this option, the entire source item graph can be copied into the destination.
+     *
+     * Can be used alone and in combination with
+     * COCopierCopiesNonCompositeReferencesExistingInDestination.
+     */
+    COCopierCopiesNonCompositeReferencesMissingInDestination = 4,
+    /**
+     * Whether items reachable through non composite references are copied, when they exist in the
```

Comment:
Copying items that exists in the destination means updating the item to match the state in the source item graph I guess?

Reply:
It depends whether you pass COCopierReusesSourceUUID. 

If you pass COCopierReusesSourceUUID, then the item in the destination is updated to match the item in the source graph.

If you don't pass  COCopierReusesSourceUUID, then the corresponding item in the destination is left untouched, because the item in the source graph is copied into the destination with a different UUID.

I'll add a note about this point to make it clearer.